src/shared/conf-parser.c

```diff
@@ -597,48 +597,6 @@ static int config_parse_many_files(
         return 0;
 }
 
-/* Parse one main config file located in one of the config paths, and its drop-ins.
- * This is what all systemd daemons are supposed to do. */
-int config_parse_config_file(
```

Comment:
Ugh, so this is removed in later commits.

Then I think it's better to avoid modifying this function but instead introduce `config_parse_standard_file_with_dropins` earlier if possible?